components/TopBar.js

```diff
@@ -67,202 +70,279 @@ const NavItem = styled(StyledLink)`
   }
 `;
 
+const MainNavItem = styled(StyledLink)`
+  color: #334155;
+  font-weight: 500;
+  font-size: 14px;
+  padding: 12px;
+  border-radius: 8px;
+  transition-property: color, background-color;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 150ms;
+  @media (hover: hover) {
+    :hover {
+      color: #0f172a;
+      background-color: #f1f5f9;
+    }
+  }
+  ${props => props.isActive && `background-color: #f1f5f9;`}
+`;
+
 const TopBar = ({ showSearch, menuItems, showProfileAndChangelogMenu }) => {
-  const [showMobileMenu, setShowMobileMenu] = useState(false);
   const [showSearchModal, setShowSearchModal] = useState(false);
+  const [showMobileMenu, setShowMobileMenu] = useState(false);
   const ref = useRef();
+  const { LoggedInUser } = useLoggedInUser();
+  const router = useRouter();
 
   // We debounce this function to avoid conflicts between the menu button and TopBarMobileMenu useGlobalBlur hook.
   const debouncedSetShowMobileMenu = debounce(setShowMobileMenu);
 
   const toggleMobileMenu = () => {
     debouncedSetShowMobileMenu(state => !state);
   };
+  const useDashboard = LoggedInUser?.hasPreviewFeatureEnabled(PREVIEW_FEATURE_KEYS.DASHBOARD);
```

Comment:
I feel like this changed so much it would probably be a good idea to just duplicate the component to `DashboardTopBar` and move the conditional to one component above.